deployment/dynatrace-azure-logs.sh

```diff
@@ -250,6 +250,21 @@ if ! command -v az &> /dev/null; then
     exit
 fi
 
+if [ $(az version -o table |awk 'NR >= 3 {print $1}') == "2.29.0" ] &> /dev/null; then
+
+    echo -e "\e[91mERROR: \e[37mVersion 2.29.0 of Azure CLI is subject to a bug which prevents Azure Log Forwarder setup from working properly."
+    echo -e "You can find the bug description under the link bellow:"
+    echo -e
```

Comment:
1. Maybe use "\n" instead of multiple "echo -e"? 
2. I would skip the explanataion part, just say that we don't support 2.29.0, please use any other and go

Reply:
1. I was trying to stick to the formatting of messages used in other parts of the script.
2. IMHO we should be upfront on why we are disallowing that specific version. I can simplify the message but I would leave the info on the problem visible.